packages/graphback-core/src/crud/mappingHelpers.ts

```diff
@@ -170,3 +170,11 @@ export function getInputFieldType(field: GraphQLField<any, any>): GraphQLInputTy
 
   return undefined;
 }
+
+/**
+ * Get the name of delta query for a model
+ * @param typeName Name of the model
+ */
+export function getDeltaQuery(typeName: string) {
+  return `sync${pluralize(typeName)}`;
```

Comment:
Why do we pluralise? I have seen it all over in the file so this is a general remark not specific for this PR.

I am wondering if the lib used takes into account complex pluralisation rules from other languages? E.g in [French](https://grammar.collinsdictionary.com/french-easy-learning/forming-plurals#:~:text=Most%20French%20nouns%20form%20their,plural%2C%20with%20a%20few%20exceptions.) the rules are so different that suffix an `s` to a business model `Hopital` (Hospital in French) will be wrong, as it should be `Hopitaux`.  I'd generally tend to stay off from doing this kind of stuff and just use the model name.

Reply:
But then you lead to misleading names in the schema like:

`findNote` instead of `findNotes`.

One implies fetching a single record, the other implies multiple.

I just checked and it seems it only supports English as a language

Reply:
I'd still like to use it, as it will work for the majority of cases.

But as you said there are cases where it simply won't.

A suggestion is to have a model annotation which can override the default resolver names:

```gql
"""
@model
@crud.find: 'trouverDesHôpitaux'
"""
```

This is something we can deal with if a request comes in, but for now sticking with pluralize will do.

Reply:
Let's keep it this way considering the naming constraints we have. I'll keep in my head and come back to this once I have a better proposition.